src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import Cell from './Cell';
+
+export default class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="">
+        <div className="text-white text-end">
+          <Cell myClass="cell bg-dark fs-3">0</Cell>
+        </div>
+        <div className="row row-cols-4">
```

Comment:
- [ ] in order to fix the linter error you asked about you can fix it by including a `constructor` method, similar to below:
```
  constructor(props) {
    super(props);
    this.state = {};
  }
```